public/locales/en/translations.json

```diff
@@ -138,7 +138,8 @@
         "deviceTypeAlerts": "Alert by device type",
         "criticalAlerts": "Critical alerts",
         "currentWindow": "Currently",
-        "previousWindow": "Previously"
+        "previousWindow": "Previously",
+        "tsiExplorerUrl": "Explore in Time Series Insights"
```

Comment:
These aren't URLs.  They're the friendly label for a URL. So something like tsiExplorerLinkLabel, but the suffix part isn't really needed (and we usually haven't put it in other translations)... so just plain tsiExplorer would be better... or maybe exploreTimeSeries.

Not sure I like using the TSI abbreviation, but won't fight too hard on it.

Reply:
Agree with the TSI abbreviation and the url label makes sense, I'll update it.